File: programmers/Lv_2/HwangJunghoon/최솟값 만들기.py
def solution(A,B):
    answer = 0
    #리스트 하나는 순서대로, 나머지는 역순으로 정렬한 후, 같은 인덱스끼리 곱한 것이 최소이다
    A.sort()
    B.sort(reverse=True)
    for i in range(len(A)):
        answer+=A[i]*B[i]
    return answer

File: programmers/Lv_2/Kimyongho/최솟값만들기.java
import java.util.*;
class Solution
{
    public int solution(int []A, int []B)
    {
        int answer = 0;
        int n1 [] = A;
        int n2 [] = B;
        int n3 [] = new int [n2.length];
        
        Arrays.sort(n1);
        Arrays.sort(n2);
        for(int i = 0; i < n3.length; i++){
            n3[i] = n2[n2.length - 1 - i];
        }
        
        for(int i = 0; i < n1.length; i++){
            answer += (n1[i] * n3[i]);
        }
        
        return answer;
    }
}
